stdlib/src/collections/list.mojo

```diff
@@ -864,3 +874,18 @@ struct List[T: CollectionElement](
 
 fn _clip(value: Int, start: Int, end: Int) -> Int:
     return max(start, min(value, end))
+
+
+fn _move_pointee_into_many_elements[
+    T: CollectionElement, //, hint_trivial_type: Bool
+](dest: UnsafePointer[T], src: UnsafePointer[T], size: Int,):
```

Comment:
No worries, thanks for bearing with me.  I should have noted this was a quibble since it's both:
- Not discussed in our coding style guide
- Not enforced by `mblack`

Both of these will change over time, and even our formatter may change heavily.  I don't feel strongly re this policy decision, by the way. :)